client/components/SettingsPane.tsx

```diff
@@ -0,0 +1,190 @@
+                        value={rateLimiterType}
+                        onChange={onRateLimiterChange}
+                    >
+                        <option disabled value="None">
+                            Select an Option
+                        </option>
+                        <option value="FIXED_WINDOW">Fixed Window</option>
+                        <option value="LEAKY_BUCKET">Leaky Bucket</option>
+                        <option value="TOKEN_BUCKET">Token Bucket</option>
+                        <option value="SLIDING_WINDOW_LOG">Sliding Window Log</option>
+                        <option value="SLIDING_WINDOW_COUNTER">Sliding Window Counter</option>
+                    </select>
+                    <Slider
+                        name="Capacity"
+                        value={capacity}
+                        displayValue={capacity}
+                        onChange={onCapacityChange}
+                        min={0}
+                        max={500}
+                        unit="tokens"
+                    />
+                    {['TOKEN_BUCKET', 'LEAKY_BUCKET'].includes(rateLimiterType) ? (
+                        <Slider
+                            name="Rate"
+                            value={refillRate}
+                            displayValue={refillRate}
+                            onChange={onRefillRateChange}
+                            min={0}
+                            max={100}
+                            unit="tokens/s"
+                        />
+                    ) : (
+                        <Slider
+                            name="Window Size"
+                            value={windowSize * 1000} // Slider goes down to millisecond for granularity
+                            displayValue={windowSize} // Slider goes down to millisecond for granularity
+                            onChange={onWindowSizeChange}
+                            min={0}
+                            max={600000} // 10 minutes
+                            unit="s"
+                        />
+                    )}
+                    <div className="panelButtonGroup">
+                        <button
+                            className="panelButton"
+                            disabled
+                            id="updateSettings"
+                            type="button"
+                            onClick={onUpdate}
+                        >
+                            Apply
+                        </button>
+                        <button className="panelButton" disabled id="resetQueries" type="button">
+                            Clear
+                        </button>
+                        <button
+                            className="panelButton"
+                            id="updateDefault"
+                            type="button"
+                            onClick={(e) => onUpdate(e, true)}
+                        >
+                            Update Project Default
```

Comment:
These button names could be more descriptive. The word Default doesn't seem right here. Maybe "Update Project Settings" indicating that they are updating the settings based on the rate limiter.

Same for reset. Maybe "Restore Project Settings"